### PR TITLE
Np 46571/np 46727 moving and renaming

### PR DESCRIPTION
**Pull request by StigNorland**

### CODE REVIEW

File: template.yaml
Comment (StigNorland) [inline template.yaml:1]: Please have a look

File: docs/openapi.yaml
Comment (StigNorland) [inline docs/openapi.yaml:1]: Please have a look

**Review by truhacevkir — APPROVED** [review]
Looks good, you can use following command to validate template.yaml:

cfn-lint -t template.yaml.  (you have to install cfn-lint)

**Comment by truhacevkir** [comment]
I would checkout branch in sandbox/dev to see if everything works as expected

**Pull request merged**